tests/feed/test_feed.py

```diff
@@ -77,3 +77,35 @@ def test_posts_order_in_dict_by_creation_date(self, num_of_posts, persist_user,
 
         posts_for_user = list(PostManager.get_posts_for_user(persist_user).keys())
         assert all(earlier.date <= later.date for earlier, later in zip(posts_for_user, posts_for_user[1:]))
+
+
+@pytest.mark.django_db
+class TestPostsListView:
+    def test_posts_list_view_loaded_authorized(self, authorized_client):
+        post_list_response = authorized_client.get('/feed/list/')
+        assert post_list_response.status_code == 200
+
+    def test_posts_list_view_loaded_unauthorized(self, client):
+        post_list_response = client.get('/feed/list/')
+        assert post_list_response.status_code == 302
+
+    def test_create_post(self, client, persist_course, persist_user, persist_first_student_course):
```

Comment:
What is the difference between this and `test_created_post_in_view`? The names look similar

Reply:
removed duplicate test